web/spacewalk-web.changes

```diff
@@ -1,4 +1,3 @@
-- Removed invalid '\r' character from Roboto*/Icon.
```

Comment:
This was maybe a bit too much? Not sure if we should remove changelog entries which are already in?

Reply:
On the other hand it is also irrelevant for the end user. @juliogonzalez ?

Reply:
It was not tagged/released, so I am OK with removing it :-)